docs/src/main/sphinx/release/release-470.md

```diff
@@ -0,0 +1,125 @@
+# Release 470 (5 Feb 2025)
+
+## General
+
+* Add [](/connector/duckdb). ({issue}`18031`)
+* Add Loki connector. ({issue}`23053`)
+* Add support for access control systems to control privileges for
+  `CREATE FUNCTION` and `DROP FUNCTION`. ({issue}`24696`)
+* Allow non-lower case variable names in SQL user defined functions. ({issue}`24460`)
+* Prevent failures of the `ARRAY_HISTOGRAM` function resulting from null values.
+  ({issue}`24765`)
+* Increase compatibility with S3-compliant storage solutions as fault-tolerant
+  execution exchange storage. ({issue}`24822`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+* {{breaking}} Raise minimum runtime requirement to Java 11. ({issue}`23639`)
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+* {{breaking}} Raise minimum runtime requirement to Java 11. ({issue}`23639`)
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Prevent connection leakage when using the Azure file system. ({issue}`24116`)
```

Comment:
Reword connection leakage .. but to what?

Reply:
to the azure blob fs